src/com/levi/Game.java

```diff
@@ -1,116 +1,86 @@
+    public Game(final Board boardOne, final Board boardTwo) {
+        this.boardOne = boardOne;
+        this.boardTwo = boardTwo;
     }
 
     /**
-     * Returns locations of player 1 ships.
-     * @return List - List of locations.
+     * Calls the method to insert your shot and method to check guess.
+     * @return Player - Winner of the game
      */
-    public List<String> getLocations1() {
-        return locations1;
-    }
+    public Player start() {
+        while (!boardOne.isAllShipDestroyed() && !boardTwo.isAllShipDestroyed()) {
+            String userGuess;
 
-    /**
-     * Returns locations of player 2 ships.
-     * @return List - List of locations.
-     */
-    public List<String> getLocations2() {
-        return locations2;
-    }
+            System.out.print(boardOne.getPlayer().getName());
+            userGuess = getUserInput();
+            boardTwo.checkShot(userGuess);
 
-    /**
-     * Returns name of player 1.
-     * @return String - Name of player 1.
-     */
-    public String getNameP1() {
-        return nameP1;
-    }
+            if (boardTwo.isAllShipDestroyed()) {
+                return boardOne.getPlayer();
+            }
 
-    /**
-     * Returns name of player 2.
-     * @return String - Name of player 2.
-     */
-    public String getNameP2() {
-        return nameP2;
+            System.out.print(boardTwo.getPlayer().getName());
+            userGuess = getUserInput();
+            boardOne.checkShot(userGuess);
+        }
+        return boardTwo.getPlayer();
     }
 
     /**
-     * Reads file and insert locations.
-     * @param file Location of file.
-     * @return List of locations.
+     * Gets guess player.
+     * @return String - Value user guess.
      */
-    public List<String> reader(String file) {
-        List<String> locations = new ArrayList<>();
-        int line = 0;
-        String temp;
-
-        try {
-            FileReader arq = new FileReader(file);
-            BufferedReader lerArq = new BufferedReader(arq);
-
-            String linha = lerArq.readLine();
-            while (linha != null) {
-
-                for (int i = 0; i < linha.length(); i++){
-                    if(linha.charAt(i) == '1'){
-                        temp = String.valueOf(alphabet.charAt(line));
-                        locations.add(temp.concat(Integer.toString(i)));
-                    }
-                }
-                linha = lerArq.readLine();
-                line++;
-            }
+    public String getUserInput() {
+        Scanner in = new Scanner(System.in);
+        System.out.println(", enter the location of your shot: ");
+        String inputLine = in.nextLine();
 
-            arq.close();
-
-        } catch (IOException e) {
-            System.err.println("Error opening file: " + e.getLocalizedMessage());
+        if (inputLine.length() == 0) {
+            return null;
```

Comment:
What occur when the getUserInfo return null?

Reply:
Updates on how to operate in commit d008c63.